tests/authentication/test_backend.py

```diff
@@ -26,7 +26,7 @@ def test_gophish_authenticate():
     auth = backend.JSONWebTokenAuthentication()
     user, token = auth.authenticate(request)
     assert user == {"username": "gophish", "groups": {"develop"}}
-    assert token == "Empty token"
+    assert token == "Empty token"  # nosec
```

Comment:
Please document the reason for the `# nosec` here, similar to other places.

Reply:
@dav3r, I put a comment at the top of the file detailing why nosec exists in multiple of the tests in this file.